### ISSUE TITLE
emulating entire 32-bit OS

### ISSUE THREAD

**Issue by hatonthecat** [ptitSeb/box86#1004]
Hi, is it possible to emulate the entire operating system, such as Windows 95, or a linux-based iso file? I'd like to load Windows 95 on a microcontroller with less than 4MB of RAM: https://github.com/hatonthecat/linux_distro_tests?tab=readme-ov-file#9-26-24-windows-95

**Comment by ptitSeb** [comment]
No, that's not how box86 has been designed. It doesn't emulatated OS-level opcodes, and is restricted to user-level emulation.
